buildSrc/src/main/groovy/MuzzlePlugin.groovy

```diff
@@ -80,21 +89,18 @@ class MuzzlePlugin implements Plugin<Project> {
           project.getLogger().info('No muzzle pass directives configured. Asserting pass against instrumentation compile-time dependencies')
           ClassLoader userCL = createCompileDepsClassLoader(project, bootstrapProject)
           ClassLoader instrumentationCL = createInstrumentationClassloader(project, toolingProject)
-          Method assertionMethod = instrumentationCL.loadClass('io.opentelemetry.javaagent.muzzle.matcher.MuzzleGradlePluginUtil')
-            .getMethod('assertInstrumentationMuzzled', ClassLoader.class, ClassLoader.class, boolean.class)
-          assertionMethod.invoke(null, instrumentationCL, userCL, true)
+          MuzzleGradlePluginUtil.@Companion.assertInstrumentationMuzzled(instrumentationCL, userCL, true)
```

Comment:
this was a bit of a weird change, but we used to use reflection to access this because it wasn't available at plugin compile time. Now it is, and in addition it was refactored to be a kotlin companion object rather than just a java static method. This refactor causes this reflection to fail, so I updated it to just access it normally.